### PR TITLE
adds set scalar content method to request information

### PR DESCRIPTION
**Pull request by baywet**
related https://github.com/microsoft/kiota/pull/1560

### CODE REVIEW

File: packages/http/fetch/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
         "test:es": "tsc -b tsconfig.es.test.json && mocha 'dist/es/test/common/**/*.js'  --require  esm && mocha 'dist/es/test/node/**/*.js'  --require  esm"
     },
     "dependencies": {
-        "@microsoft/kiota-abstractions": "1.0.0-preview.4",
+        "@microsoft/kiota-abstractions": "^1.0.0-preview.5",
```
Comment (nikithauc) [inline packages/http/fetch/package.json:47]:
```suggestion
        "@microsoft/kiota-abstractions": "1.0.0-preview.5",
```

Comment (nikithauc) [reply packages/http/fetch/package.json:47]: The preview version will be locked to a specific version of abstractions. 

The changes in a higher version of abstractions may/may not be compatible.

File: packages/serialization/json/package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     },
     "homepage": "https://github.com/microsoft/kiota-typescript#readme",
     "dependencies": {
-        "@microsoft/kiota-abstractions": "1.0.0-preview.4",
+        "@microsoft/kiota-abstractions": "^1.0.0-preview.5",
```
Comment (nikithauc) [inline packages/serialization/json/package.json:43]:
```suggestion
        "@microsoft/kiota-abstractions": "1.0.0-preview.5",
```

File: packages/abstractions/src/requestInformation.ts [hunk 3]
```diff
@@ -104,25 +107,95 @@ export class RequestInformation {
...
       .getSerializationWriter(contentType);
+  };
+  /**
+   * Sets the request body from a model with the specified content type.
+   * @param values the scalar values to serialize.
+   * @param contentType the content type.
+   * @param requestAdapter The adapter service to get the serialization writer from.
+   * @typeParam T the model type.
+   */
+  public setContentFromScalar = <T>(
```
Comment (nikithauc) [inline packages/abstractions/src/requestInformation.ts:155]: Please add unit tests validating this change.

Comment (baywet) [reply packages/abstractions/src/requestInformation.ts:155]: added

File: packages/serialization/json/package.json (hunk 2 above)
Comment (baywet) [reply packages/serialization/json/package.json]: reverted (across packages)

**Review by nikithauc — APPROVED** [review]

**Pull request merged**